ui/component/viewers/videoViewer/internal/videojs.jsx

```diff
@@ -376,6 +366,27 @@ export default React.memo<Props>(function VideoJs(props: Props) {
         vjsPlayer = window.player;
       }
 
+      // hide unused elements on livestream
+      if (isLivestream) {
+        vjsPlayer.addClass('vjs-live');
+        vjsPlayer.addClass('vjs-liveui');
+        // $FlowIssue
+        vjsPlayer.controlBar.currentTimeDisplay?.el().style.setProperty('display', 'none', 'important');
+        // $FlowIssue
+        vjsPlayer.controlBar.timeDivider?.el().style.setProperty('display', 'none', 'important');
+        // $FlowIssue
+        vjsPlayer.controlBar.durationDisplay?.el().style.setProperty('display', 'none', 'important');
+      } else {
+        vjsPlayer.removeClass('vjs-live');
+        vjsPlayer.removeClass('vjs-liveui');
+        // $FlowIssue
+        vjsPlayer.controlBar.currentTimeDisplay?.el().style.setProperty('display', 'block', 'important');
+        // $FlowIssue
+        vjsPlayer.controlBar.timeDivider?.el().style.setProperty('display', 'block', 'important');
+        // $FlowIssue
+        vjsPlayer.controlBar.durationDisplay?.el().style.setProperty('display', 'block', 'important');
+      }
```

Comment:
- [ ] Controlbar broken in chromecast

I don't recall the exact sequence, but I was just casting livestreams one after another, and this happened:

<img width="684" alt="image" src="https://user-images.githubusercontent.com/64950861/176161338-431c69ed-4a14-4770-92b0-7ae79766f3eb.png">